src/mono/mono.proj

```diff
@@ -27,6 +27,8 @@
     <LibrariesTesthostRoot>$(ArtifactsDir)bin\testhost\$(NetCoreAppCurrent)-$(TargetOS)-$(LibrariesTestConfig)-$(Platform)\</LibrariesTesthostRoot>
     <LibrariesTesthostRuntimeDir>$(LibrariesTesthostRoot)shared\Microsoft.NETCore.App\$(ProductVersion)\</LibrariesTesthostRuntimeDir>
     <XcodeDir Condition="'$(XcodeDir)' == ''">/Applications/Xcode.app/Contents/Developer</XcodeDir>
+    <_BuildMonoAOTCrossCompiler Condition="'$(TargetsiOS)' == 'true' and '$(TargetsiOSSimulator)' != 'true'">true</_BuildMonoAOTCrossCompiler>
+    <_BuildMonoAOTCrossCompiler Condition="'$(TargetstvOS)' == 'true' and '$(TargetstvOSSimulator)' != 'true'">true</_BuildMonoAOTCrossCompiler>
```

Comment:
nit: I originally thought that we should add this in a new PropertyGroup inside of the `BuildMonoRuntimeUnix` target but this is actually better since we can reuse it to simplify the condition in the `<!-- AOT cross-compiler specific options -->`.

We should do that and remove the underscore prefix.